ffmpeg/change-cmake-version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 913a303..ef5fba9 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -9,7 +9,7 @@
+ # PATENTS file, you can obtain it at https://www.aomedia.org/license/patent-license.
+ #
+ 
+-cmake_minimum_required(VERSION 3.16)
++cmake_minimum_required(VERSION 3.13.4)
```

Comment:
Let's use a sed command for small changes like this:
https://github.com/bytedeco/javacpp-presets/blob/master/onnxruntime/cppbuild.sh#L69